addons/website_form/controllers/main.py

```diff
@@ -19,8 +20,13 @@
 class WebsiteForm(http.Controller):
 
     # Check and insert values from the form on the model <model>
-    @http.route('/website_form/<string:model_name>', type='http', auth="public", methods=['POST'], website=True)
+    @http.route('/website_form/<string:model_name>', type='http', auth="public", methods=['POST'], website=True, csrf=False)
     def website_form(self, model_name, **kwargs):
+        if request.session.uid:
+            if not request.validate_csrf(kwargs.get('csrf_token')):
+                raise BadRequest('Session expired (invalid CSRF token)')
+        else:
+            request.params.pop('csrf_token', False)
```

Comment:
need this extra line to avoid that csrf are automatically appended as custom field in the description field of the record.